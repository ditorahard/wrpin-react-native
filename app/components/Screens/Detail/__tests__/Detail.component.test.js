import 'react-native';
import React from 'react';
import DetailScreen from '../Detail.component';
import renderer from 'react-test-renderer';

it('Detail renders correctly', () => {
  const tree = renderer.create(
    <DetailScreen />
    ).toJSON();
  expect(tree).toMatchSnapshot();
});