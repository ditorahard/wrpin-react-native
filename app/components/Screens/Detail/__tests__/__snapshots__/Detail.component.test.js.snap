// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Detail renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
        "height": 1334,
      },
      Object {
        "padding": 20,
      },
    ]
  }
>
  <RCTScrollView
    automaticallyAdjustContentInsets={false}
    contentContainerStyle={
      Array [
        Object {
          "padding": undefined,
        },
        undefined,
      ]
    }
    contentInset={
      Object {
        "bottom": 0,
      }
    }
    enableAutomaticScroll={true}
    enableOnAndroid={false}
    enableResetScrollToCoords={true}
    extraHeight={75}
    extraScrollHeight={0}
    getScrollResponder={[Function]}
    handleOnScroll={[Function]}
    keyboardDismissMode="interactive"
    keyboardOpeningTime={250}
    keyboardShouldPersistTaps="handled"
    keyboardSpace={0}
    onScroll={[Function]}
    resetKeyboardSpace={[Function]}
    resetScrollToCoords={
      Object {
        "x": 0,
        "y": 0,
      }
    }
    scrollEventThrottle={1}
    scrollForExtraHeightOnAndroid={[Function]}
    scrollIntoView={[Function]}
    scrollToEnd={[Function]}
    scrollToFocusedInput={[Function]}
    scrollToPosition={[Function]}
    showsVerticalScrollIndicator={true}
    style={
      Object {
        "backgroundColor": "transparent",
        "flex": 1,
      }
    }
    update={[Function]}
    viewIsInsideTabBar={false}
  >
    <View>
      <ActivityIndicator
        animating={true}
        color="#45D56E"
        hidesWhenStopped={true}
        size="large"
        style={
          Object {
            "height": 80,
          }
        }
      />
    </View>
  </RCTScrollView>
</View>
`;
